content/en/cosign/kubernetes.md

```diff
@@ -4,7 +4,7 @@ category: "Cosign"
 position: 102
 ---
 
-## Kubernetes Secrets
+## Kubernetes Secrets  remove this page
```

Comment:
Thanks @jonvnadelberg , the 'remove this' sections, are they todo's or something that is picked up programmatically (I can't confess to know the docs code to well).

Reply:
Yes, there were issues with the pull request, that caused me to close it and redo it.   This is one of them.